docs/src/languages/java/migration.md

```diff
@@ -0,0 +1,166 @@
+
+### Route
+
+```yaml
+"https://{default}/":
+  type: upstream
+  upstream: 1
+"https://www.{default}/":
+  type: redirect
+  to: "https://{default}/"
+```
+
+1. It defines the application will link in the route, e.g.: `"app:http"`
+
+{{< note >}}
+
+Be aware that the container has a maximum memory if the application is stateless to increase a more powerful computer, there is the strategy to use a [load balancer](https://community.platform.sh/t/how-to-configure-load-balancer-in-a-single-application/553).
+
+Be aware that after the build, it creates a read-only system. You have the [mount option to create a writable folder](https://docs.platform.sh/configuration/app/storage.html#mounts).
+
+{{< /note >}}
+
+## Microservices
+
+You have the option to use several languages in microservices. If you're using Java there are several options to aggregate these services into a microservices:
+
+* [Maven Modules](https://maven.apache.org/guides/mini/guide-multiple-modules.html)
+* [Gradle Multi-project](https://guides.gradle.org/creating-multi-project-builds/)
+* [Git submodules](https://docs.platform.sh/development/submodules.html)
+
+[Platform.sh supports multiple applications](https://docs.platform.sh/configuration/app/multi-app.html) and there are two options:
+
+* [One application YAML file to each application](https://docs.platform.sh/configuration/app.html)
+* [Aggregate all applications in a single file with applications.yaml](https://docs.platform.sh/configuration/app/multi-app.html#applicationsyaml)
+
+| Article                                                      | Content                                                      |
+| ------------------------------------------------------------ | ------------------------------------------------------------ |
+| [Microservices in the cloud, part two](https://platform.sh/blog/2019/microservices-in-the-cloud-part-two/) | [Source](https://github.com/EventosJEspanol/latin-america-micro-profile) |
+| [Microservices in the cloud, part one](https://platform.sh/blog/2019/microservices-in-the-cloud-part-one/) | [Source](https://github.com/EventosJEspanol/latin-america-micro-profile) |
+| [Multiple Applications](https://community.platform.sh/t/multiple-applications-tomcat/468) | [Source](https://github.com/platformsh-examples/tomcat-multi-app) |
+| [Configure multi-applications with applications.yaml](https://community.platform.sh/t/how-to-configure-multi-applications-with-applications-yaml/552) | [Source](https://github.com/platformsh-examples/tomcat-multi-app-applications) |
+
+{{< note >}}
+
+As a single application, you have the option to set load balancer to some or [all applications in the project cluster](https://community.platform.sh/t/how-to-configure-load-balancer-in-a-multiple-applications/554).
```

Comment:
This is technically true for multi-app, as well.  It's just more involved.

Reply:
Isn't this redundant with the note above that I just commented on?

Reply:
Yes, that is true to multi-app. I would like to keep it redudant